monai/bundle/workflows.py

```diff
@@ -44,12 +44,14 @@ class BundleWorkflow(ABC):
         workflow_type: specifies the workflow type: "train" or "training" for a training workflow,
             or "infer", "inference", "eval", "evaluation" for a inference workflow,
             other unsupported string will raise a ValueError.
-            default to `train` for train workflow.
+            default to `None` for only using meta properties.
         workflow: specifies the workflow type: "train" or "training" for a training workflow,
             or "infer", "inference", "eval", "evaluation" for a inference workflow,
             other unsupported string will raise a ValueError.
             default to `None` for common workflow.
-        properties_path: the path to the JSON file of properties.
+        properties_path: the path to the JSON file of properties. If workflow type is specified, the properties
+            will be loaded from the file based on the workflow type and meta, otherwise, the properties will be loaded from
+            "train". If the file does not exist, the default properties will be used.
```

Comment:
We should explain here what these properties are, that is they specify what the components of the bundle are such as the definitions that would be found in the config files for a config workflow. We should mention what the default properties are, that is it's those in `monai/bundle/properties.py`.

Reply:
Update the docstring, please help review it again, thanks.